xsl/webhelp-common.xsl

```diff
@@ -525,6 +525,15 @@ border: none; background: none; font-weight: none; color: none; }
                 </div>
 
                 <xsl:call-template name="user.footer.navigation"/>
+                <xsl:choose><xsl:when test="$webhelp.include.search.tab = '0'">
```

Comment:
Shouldn't this be the opposite? That is, test if equal to 1?

I'm not big on XSLT so I may just be confused... but in the Makefile above, you set this variable to 1 to include the search box, and here you seem to be including it when it's 0...